libkatana_python_native/src/EntityTypeManager.cpp

```diff
@@ -112,6 +112,15 @@ katana::python::InitEntityTypeManager(py::module_& m) {
                 self, KATANA_CHECKED(
                           self->GetOrAddNonAtomicEntityType(set_of_type_ids))};
           })
+      .def(
+          "get_or_add_non_atomic_entity_type_from_strings",
+          [](katana::EntityTypeManager* self,
+             std::set<std::string> type_strs) -> Result<EntityType> {
+            return EntityType{
+                self,
+                KATANA_CHECKED(
+                    self->GetOrAddNonAtomicEntityTypeFromStrings(type_strs))};
+          })
```

Comment:
Don't add this. It was left out intentionally because it can be written trivially in Python like this `m.get_or_add_non_atomic_entity_type({m[n] for n in string_container)`.

Reply:
The method `GetOrAddNonAtomicEntityTypeFromStrings` just converts each argument to a type and then gets the intersection of them (the type that has all the given types). So you can just do that yourself with a comprehension. The only reason `GetOrAddNonAtomicEntityTypeFromStrings` exists in C++ is that collections handling is so bad in C++ that writing it out would be really painful.